src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1050,6 +1088,11 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
 
         if (event.phase == TickEvent.Phase.START)
         {
+            if(!(this.citizens.size() < CITIZEN_MINIMUM_FOR_RAID))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the opposite operator (">=") instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1940)